lua/gk.lua

```diff
@@ -15,8 +15,39 @@ return function (net_conf, numa_table)
 	local ggu_lcore = table.remove(gk_lcores)
 	gatekeeper.gk_assign_lcores(gk_conf, gk_lcores)
 
+	gk_conf.max_num_ipv4_rules = 1024
+	gk_conf.num_ipv4_tbl8s = 256
+	gk_conf.max_num_ipv6_rules = 1024
+	gk_conf.num_ipv6_tbl8s = 65536
+
+	local fib_entries = {
+		{
+			["ip_prefix"] = "187.73.30.0/30",
+			["action"] = gatekeeper.c.GK_FWD_GRANTOR,
+			["grantor"] = "10.0.0.3",
+		},
+
+		{
+			["ip_prefix"] = "187.73.31.0/30",
+			["action"] = gatekeeper.c.GK_FWD_GRANTOR,
+			["grantor"] = "3ffe:2501:200:3::1",
+			["gateway"] = "10.0.0.254",
```

Comment:
Just to confirm: in the FIB entries, the Grantor server and gateway should have the same IP version. With the pseudo-FIB-entry ``["ip_prefix"] = "187.73.31.0/30",
["action"] = gatekeeper.c.GK_FWD_GRANTOR,
["grantor"] = "128.197.41.100",
["gateway"] = "10.0.0.254",
["gw_prefix"] = "10.0.0.1/24",``, we need to split it into two entries, i.e.,

Entry 1: ``["ip_prefix"] = "187.73.31.0/30",
["action"] = gatekeeper.c.GK_FWD_GRANTOR,
["grantor"] = "128.197.41.100",``

Entry 2: ``["ip_prefix"] = "128.197.41.100/32", ["action"] = gatekeeper.c.GK_FWD_GATEWAY, ["gateway"] = "10.0.0.254", ["gw_prefix"] = "10.0.0.1/24",``.

Is this right?